src/app/store/reducers/VehicleTestResultModel.reducers.ts

```diff
@@ -1,20 +1,61 @@
-import { initialVehicleTestResultModelState } from '../../store/state/VehicleTestResultModel.state';
-import { EVehicleTestResultModelActions, VehicleTestResultModelActions } from '../actions/VehicleTestResultModel.actions';
+import { initialVehicleTestResultModelState } from '@app/store/state/VehicleTestResultModel.state';
+import {
+  EVehicleTestResultModelActions, SetSelectedTestResultModelSuccess,
+  VehicleTestResultModelActions
+} from '../actions/VehicleTestResultModel.actions';
 
-export function VehicleTestResultModelReducers(state = initialVehicleTestResultModelState, action: VehicleTestResultModelActions) {
+export function VehicleTestResultModelReducers(
+  state = initialVehicleTestResultModelState,
+  action: VehicleTestResultModelActions
+) {
   switch (action.type) {
     case EVehicleTestResultModelActions.GetVehicleTestResultModelSuccess: {
       return {
         ...state,
-        selectedTestResultModel: action.payload,
-        error: null // clear error message
+        vehicleTestResultModel: action.payload,
+        error: null
```

Comment:
what is this error object? Shouldn't be part of the root reducer instead.
This will need to be moved